app/src/main/java/com/projectit3048c/MainActivity.kt

```diff
@@ -43,10 +43,11 @@ class MainActivity : ComponentActivity() {
                 // A surface container using the 'background' color from the theme
                 Surface(
                     modifier = Modifier.fillMaxWidth(),
-                    color = MaterialTheme.colors.background) {
-                    CalorieFacts("Android")
-                }
-                var foo = foods
+                    color = MaterialTheme.colors.background
+                ) {
+                        CalorieFacts("Android," , foods)
+                    }
```

Comment:
Reinstated variable into the function rather than adding it to the end for better readability.